core/eth/tx.go

```diff
@@ -591,19 +591,20 @@ func (t *Transactor) GetQuorumBitmapForOperatorsAtBlockNumber(ctx context.Contex
 	// Get bitmaps in N RPCs, but in parallel.
 	type BitmapOrError struct {
 		bitmap *big.Int
-		index  uint32
+		index  int
 		err    error
 	}
 	resultChan := make(chan BitmapOrError, len(indices))
 	pool := workerpool.New(maxNumWorkerPoolThreads)
 	for i, index := range indices {
-		idx := index
+		i := i
+		index := index
```

Comment:
`i`, `index` are pretty confusing.
Can we call `indices` something like `quorumIndices` and leave a comment saying each element in `quorumIndices` corresponds to the quorum bitmap index of each operator in `byteOperatorIds` in the same order?

Reply:
done